internal/cli/atlas/backup/compliancepolicy/policies/create/create.go

```diff
@@ -0,0 +1,152 @@
+// Copyright 2023 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package create
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/mongodb/mongodb-atlas-cli/internal/cli"
+	"github.com/mongodb/mongodb-atlas-cli/internal/config"
+	"github.com/mongodb/mongodb-atlas-cli/internal/file"
+	"github.com/mongodb/mongodb-atlas-cli/internal/flag"
+	store "github.com/mongodb/mongodb-atlas-cli/internal/store/atlas"
+	"github.com/mongodb/mongodb-atlas-cli/internal/usage"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	atlasv2 "go.mongodb.org/atlas-sdk/v20230201004/admin"
+)
+
+type Opts struct {
+	cli.GlobalOpts
+	cli.WatchOpts
+	store  store.CompliancePolicyPoliciesCreator
+	fs     afero.Fs
+	path   string
+	policy *atlasv2.DataProtectionSettings
+}
+
+const (
+	active = "ACTIVE"
+)
+
+var updateTemplate = `Your backup compliance policy is being updated with the following policies:
+ID	FREQUENCY INTERVAL	FREQUENCY TYPE	RETENTION
+{{- range .ScheduledPolicyItems}}
+{{.Id}}	{{if eq .FrequencyType "hourly"}}{{.FrequencyInterval}}{{else}}-{{end}}	{{.FrequencyType}}	{{.RetentionValue}} {{.RetentionUnit}}
+{{- end}}
+{{if .OnDemandPolicyItem}}{{.OnDemandPolicyItem.Id}}	-	{{.OnDemandPolicyItem.FrequencyType}}	{{.OnDemandPolicyItem.RetentionValue}} {{.OnDemandPolicyItem.RetentionUnit}}{{end}}
+`
+var updateWatchTemplate = `Your backup compliance policy has been updated with the following policies:
+ID	FREQUENCY INTERVAL	FREQUENCY TYPE	RETENTION
+{{- range .ScheduledPolicyItems}}
+{{.Id}}	{{if eq .FrequencyType "hourly"}}{{.FrequencyInterval}}{{else}}-{{end}}	{{.FrequencyType}}	{{.RetentionValue}} {{.RetentionUnit}}
+{{- end}}
+{{if .OnDemandPolicyItem}}{{.OnDemandPolicyItem.Id}}	-	{{.OnDemandPolicyItem.FrequencyType}}	{{.OnDemandPolicyItem.RetentionValue}} {{.OnDemandPolicyItem.RetentionUnit}}{{end}}
+`
+var example = `How to run atlas backups compliancepolicy policies create with --file.
+As an example, the file should be in the following format:
+
+{
+	"frequencyInterval": 1,
+	"frequencyType": "daily",
+	"id": "stringstringstringstring",
+	"retentionUnit": "days",
+	"retentionValue": 0
+}
+
+To get the ID of a policy item, run "atlas backups compliancepolicy policies describe".
+
+Finally, run the command as such: "atlas backups compliancepolicy policies update --file /path/to/file"
+`
```

Comment:
this example may break when generating docs in the web where spaces are meaningful for snooty the rendering tool
 please keep examples on how to run the command and leave out the file example, check the cluster command for examples on how we handle this were the definition of the file is linked in the file help